cmd/cogito/main_test.go

```diff
@@ -62,14 +70,45 @@ func TestRunPutSmokeIntegrationSuccess(t *testing.T) {
 	var out bytes.Buffer
 	var logOut bytes.Buffer
 	inputDir := testhelp.MakeGitRepoFromTestdata(t, "../../cogito/testdata/one-repo/a-repo",
-		testhelp.HttpsRemote("the-owner", "the-repo"), "dummySHA", "dummyHead")
+		testhelp.HttpsRemote("the-owner", "the-repo"), "dummySHA", wantGitRef)
+	t.Setenv("COGITO_GITHUB_API", ts.URL)
 
 	err := run(in, &out, &logOut, []string{"out", inputDir})
 
 	assert.NilError(t, err, "\nout: %s\nlogOut: %s", out.String(), logOut.String())
+	ts.Close() // Avoid races before following asserts.
+	assert.Equal(t, ghReq.State, string(wantState), "", ghReq.State, string(wantState))
+	assert.Equal(t, path.Base(URL.Path), wantGitRef)
 }
 
-func TestRunSmokeFailure(t *testing.T) {
+func TestRunPutSuccessIntegration(t *testing.T) {
+	if testing.Short() {
```

Comment:
This is set when the `-short` flag is passed to `go test`.
Compare the Task targets `test:unit` and `test:all` to see how it is used.
File CONTRIBUTING also explains this.